frontend/src/pages/GamePage.tsx

```diff
@@ -1,25 +1,26 @@
-import React, { useState } from "react";
+// import { quizQuestions } from "./questions";
+
+import React, { useEffect, useState } from "react";
 import Terminal, { ColorMode, TerminalOutput } from "react-terminal-ui";
-import { quizQuestions } from "./questions";
+import axios from "axios";
 
 const GamePage: React.FC = () => {
-    const [terminalLineData, setTerminalLineData] = useState([
+    const [terminalLineData, setTerminalLineData] = useState<React.ReactNode[]>([
         <TerminalOutput>Welcome to the Life Choices Game!</TerminalOutput>,
         <br />,
-        <TerminalOutput>Your initial stats: Health: 50, Happiness: 50, Money: 50.</TerminalOutput>,
-        <br />,
-        <TerminalOutput>{`Question 1: ${quizQuestions[0].question}`}</TerminalOutput>,
-        ...quizQuestions[0].options.map((option, index) => (
-            <TerminalOutput key={`tout-${index}`}>{`${index + 1}. ${option.text}`}</TerminalOutput>
-        )),
+        <TerminalOutput>Loading questions, please wait...</TerminalOutput>,
     ]);
+
+    const [quizQuestions, setQuizQuestions] = useState<any[]>([]);
```

Comment:
Checkout `/validators/user.ts` and create a new schema just as defined in the postman into the file `/validators/question.ts` and replace any with that schema name. Refer [zod docs](https://zod.dev/)

Reply:
For more info, try not to use any in typescript, unless it is really really necessary. Can read articles about why not